tests/react/vanilla-utils/atomWithStorage.test.tsx

```diff
@@ -353,20 +353,49 @@ describe('atomWithStorage (in non-browser environment)', () => {
   }
 
   const addEventListener = window.addEventListener
+  const localStorage = window.localStorage
+  const sessionStorage = window.sessionStorage
 
   beforeAll(() => {
     ;(window as any).addEventListener = undefined
+    Object.defineProperty(window, 'localStorage', {
+      get() {
+        throw new Error('localStorage is not available.')
+      },
+    })
+    Object.defineProperty(window, 'sessionStorage', {
+      get() {
+        throw new Error('localStorage is not available.')
+      },
+    })
   })
 
   afterAll(() => {
     window.addEventListener = addEventListener
+    Object.defineProperty(window, 'localStorage', {
+      get() {
+        return localStorage
+      },
+    })
```

Comment:
This doesn't really restore, but maybe it's fine.

Reply:
you're right. I removed this part of the code and nothing changes. doesn't it affect using the `localStorage` in other tests?

Reply:
It may. the best is to leave it for now.